pod/perl5360delta.pod

```diff
@@ -0,0 +1,1292 @@
+=encoding utf8
+
+=head1 NAME
+
+perldelta - what is new for perl v5.36.0
+
+=head1 DESCRIPTION
+
+This document describes differences between the 5.34.0 release and the 5.36.0
+release.
+
+=head1 Notice
+
+With this release, the experimental C<switch> feature, present in every feature
+bundle since they were introduced in v5.10, has been removed from the v5.36
+bundle.  If you want to live dangerously and use it, you'll have to enable it
+explicitly.
+
+=head1 Core Enhancements
+
+=head2 C<use v5.36> enables warnings
+
+All warnings are enabled automatically within the scope of
+a C<L<use v5.35|perlfunc/use VERSION>> (or higher) declaration.
+
+=head2 C<use v5.36> now disables bareword_filehandles, indirect, and multidimensional
+
+The C<:5.36> bundle, loaded by C<use v5.36>, no longer includes
+C<bareword_filehandles>, C<indirect>, or C<multidimensional> meaning that these
+discouraged forms of syntax will not be allowed within the scope of that
+version declaration.
+
+=head2 C<defer> blocks
+
+This release adds support for C<defer> blocks, which are blocks of code
+prefixed by the C<defer> modifier. They provide a section of code which runs
+at a later time, during scope exit.
+
+For more information, see L<perlsyn/"defer blocks">.
+
+=head2 Unicode 14.0 is supported
+
+See L<https://www.unicode.org/versions/Unicode14.0.0/> for details.
+
+=head2 SIGFPE no longer deferred
+
+Floating-point exceptions are now delivered immediately, in the same way
+as other "fault"-like signals such as SIGSEGV. This means one has at
+least a chance to catch such a signal with a C<$SIG{FPE}> handler, e.g.
+so that C<die> can report the line in perl that triggered it.
+
+=head2 Stable boolean tracking
+
+The "true" and "false" boolean values, often accessed by constructions like
+C<!!0> and C<!!1>, as well as being returned from many core functions and
+operators, now remember their boolean nature even through assignment into
+variables. The new function C<isbool()> in L<Scalar::Util> can check whether
```

Comment:
This is no longer true. The function is now `builtin::is_bool`.

Reply:
Oh, thanks!  I got is_bool elsewhere, but this snuck past.